homeassistant/components/deconz/__init__.py

```diff
@@ -35,12 +35,14 @@
     if not await gateway.async_setup():
         return False
 
-    hass.data[DOMAIN][gateway.bridgeid] = gateway
+    if config_entry.unique_id is None:
```

Comment:
Add comment that it's for backwards compat.

Reply:
Will do